playbooks/robusta_playbooks/volume_analysis.py

```diff
@@ -0,0 +1,76 @@
+from robusta.api import action, PersistentVolumeEvent, RobustaPod, ObjectMeta, PodSpec, Volume, PersistentVolumeClaimVolumeSource, Container, VolumeMount, List, BaseBlock, MarkdownBlock,Finding,FindingSource,FindingType
+
+
+@action
+def volume_analysis(event: PersistentVolumeEvent):
+    # This function creates a reader pod (using the builtin hikaru library) that mount and output contents present on your persistent volume.
+    """
+    This action shows you the files present on your persistent volume
+    """
+
+    # Get persistent volume data the object contains data related to PV like metadata etc
+    volume = event.get_persistentvolume()
+
+    # Define a reader (busybox) pod which mounts the volume
+
+    reader_pod = RobustaPod(
+        apiVersion="v1",
+        kind="Pod",
+        metadata=ObjectMeta(
+            name="volume-inspector",
```

Comment:
I think you can also add the namespace here: namespace=volume.spec.claimRef.namespace,